src/pages/components/map-related/MapLayers/BusToolTip.tsx

```diff
@@ -39,7 +47,7 @@ export function BusToolTip({ position, icon }: BusToolTipProps) {
         <>
           <header className="header">
             <h1 className="title">
-              {t('line')} :<span>{siriRide && siriRide.gtfsRouteRouteShortName}</span>
+              {t('line')} :<span>{LinkToProfile}</span>
```

Comment:
```suggestion
              {t('line')} :<span>
                  <Link to={`/profile/${siriRide.gtfsRouteRouteShortName}`}>
                    {siriRide && siriRide.gtfsRouteRouteShortName}{' '}
                  </Link>
              </span>
```
isn't it easier?

Reply:
What if the siriRide is undefined?
I think that at some point of the renders it doesn't got any properties or even a value.
I can take care of the null/undefined scenario when using the **Link**  component in an inline JSX, but I personally  just like it less & have no problem to follow the project standard style.

Reply:
What about turning it into a component?

Reply:
I think that This is a good choice. I'll add it as a local Component.

Reply:
Actually, if the `siriRide` is `undefined`, we should probably hide the entire page. Don't you think?

Reply:
Yes there is no reason to show the tooltip without data. In the case I'll return NULL then there will be a white line which is the leaflet popup wrapper. So in a future issue it might be a nice to deal with it. For now it should be a good solution.

Reply:
I think it would be for the best if you'll place the link inline, and the nullish logic will be handled in outer scope